Fixing crash and Issue 9

graetzer opened the pull request:
- Fixes the [Issue of upscaled images](https://github.com/njdehoog/NHBalancedFlowLayout/issues/22) wich occurs if the partition algorithm assigns a single item to one row.
- Incorporates a fix for a crash on `[UICollectionView insertSections:]` from [Pull Request #24](https://github.com/njdehoog/NHBalancedFlowLayout/pull/24)
- It implements swipe in and out animations, when inserting or removing cells.

(Last one should be optional, but I didn't have time to implement that yet)

graetzer committed:
Incorporate all fixes

graetzer committed:
Update README.md

graetzer committed:
Better way to merge rows

njdehoog commented:
@graetzer Thanks for your PR. I am no longer maintaining this project. Would you mind if I just point people to your fork in the README?

graetzer commented:
Sure I think that's the best idea, if you don't want to test and merge the code.

njdehoog commented:
Honestly it's been too long since I looked at this code, and I don't have the time to maintain it at the moment. I updated the REAME. Thanks again.

chadpod committed:
Update NHLinearPartition.m

Change copy to mutableCopy so a NSMutableArray is returned instead of a NSArray, honoring the method signature.

graetzer committed:
Merge pull request #1 from chadpod/patch-1

Update NHLinearPartition.m

graetzer committed:
Fix crash with only 1 entry in partition array

graetzer committed:
Update NHBalancedFlowLayout.m
